Extensions/quick_tags.js

```diff
@@ -297,9 +302,6 @@ XKit.extensions.quick_tags = new Object({
 		$("#xkit-quick-tags-window").remove();
 		$("body").append(m_html);
 
-		$("#xkit-tag-input").bind("focus", XKit.extensions.quick_tags.cancel_menu_close);
-		$("#xkit-tag-input").bind("blur", XKit.extensions.quick_tags.menu_close);
```

Comment:
I understand the removal of the focus event, but doesn't removing the blur event mean that clicking outside of the quick tags menu to blur the input will no longer close the quick tags menu?